src/ui/menu.c

```diff
@@ -526,6 +526,8 @@ void mudclient_menu_item_click(mudclient *mud, int i) {
 }
 
 void mudclient_create_top_mouse_menu(mudclient *mud) {
+    mud->menu_text_position = "top"; /* TODO: extract this to a user-changeable option*/
```

Comment:
This assignment makes the string read-only so it should be declared const.

Reply:
Yeah it only matters since a string is an array (you can't write into "this"), menus are ints.

Reply:
I'm not sure how to get it to work as a const then, I'm still quite inexperienced with C.